src/main/java/nl/tudelft/contextproject/gui/PreviewController.java

```diff
@@ -370,10 +400,11 @@ private int checkShots(Shot shot) {
     private void showShot(Shot shot) {
```

Comment:
This method is still really ugly, but that wasn't the point of this branch. So I will ignore it for now.